presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotPushdownUtils.java

```diff
@@ -165,7 +164,7 @@ public ExpressionType getExpressionType()
                 if (aggregation.getMask().isPresent()) {
                     // This block handles the case when a distinct aggregation is present in addition to another aggregation function.
                     // E.g. `SELECT count(distinct COL_A), sum(COL_B) FROM myTable` to Pinot as `SELECT distinctCount(COL_A), sum(COL_B) FROM myTable`
-                    if (aggregation.getCall().getDisplayName().equalsIgnoreCase(COUNT_FUNCTION_NAME) && aggregation.getMask().get().getName().equalsIgnoreCase(aggregation.getArguments().get(0) + DISTINCT_MASK)) {
+                    if (aggregation.getCall().getDisplayName().equalsIgnoreCase(COUNT_FUNCTION_NAME)) {
```

Comment:
can we do a substring check for a distinct mask? I am not sure what else other than distinct can reach here, just want to make sure that it is indeed a distinct that we are pushing down

Reply:
ic, sure, i can add a contains check to ensure there is a `$distinct` here.